app/Mail/NewReport.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class NewReport extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct(
+        public string $name,
+        public string $email,
+        public string $phone,
+    )
+    {
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this
+            ->subject('New Report!')
+            ->view('emails.new_report');
```

Comment:
шаблон не забудь переименовать
```suggestion
            ->view('emails.new_contact');
```